proposals/src/proposal/cosmwasm/rescue_tokens.rs

```diff
@@ -0,0 +1,217 @@
+//! Rescue Tokens Proposal.
+use crate::ProposalHeader;
+
+/// Rescue Tokens Proposal.
+///
+/// The `RescueTokensProposal` rescues tokens from the treasury to a specified
+/// `to` address.
+/// The format of the proposal looks like this:
+///
+/// ```text
+/// ┌────────────────────┬───────────────────┬────────────────────┬────────────┐
+/// │                    │                   │                    │            │
+/// │ ProposalHeader 40B │ TokenAddress 20B  │ ToTokenAddress 20B │ Amount 32B │
```

Comment:
`ToTokenAddress` should be `Recipient`

Reply:
Can you also change it in the EVM side please.

Reply:
Yes, I can handle that change.
This is the PR I created for above recommend.
https://github.com/webb-tools/webb-rs/pull/65